socorro/unittest/external/boto/test_upload_telemetry_schema.py

```diff
@@ -2,30 +2,16 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from moto import mock_s3_deprecated
-
 from socorro.external.boto.upload_telemetry_schema import UploadTelemetrySchema
 from socorro.unittest.external.boto import get_config
 
 
 class TestUploadTelemetrySchema:
-    @mock_s3_deprecated
-    def test_bucket_not_found(self, caplogpp):
-        # If the bucket isn't found, the script should tell the user and return
-        # a non-zero exit code
-        config = get_config(UploadTelemetrySchema)
-        app = UploadTelemetrySchema(config)
-
-        assert app.main() == 1
-        recs = [rec.message for rec in caplogpp.records]
-        assert "Failure: The crashstats S3 bucket must be created first." in recs
```

Comment:
I nixed a lot of the "let's do special things if the bucket isn't there". It's an edge case that only happens when setting things up or if you've misconfigured the bucket name. In all those cases, it should be sending something to Sentry or the logs. It should be really obvious without additional special code we have to maintain that the bucket is either misconfigured or not there.